### PR TITLE
add darwin-23 to platforms in gemfile.lock

### PR DESCRIPTION
**Pull request by aliuk2012**
### What problem does this pull request solve?

Running `bundle install` on main modified the gemfile.lock by adding this missing platform.

Trello card: 

### Things to consider when reviewing

- Ensure that you consider the wider context.
- Does it work when run on your machine?
- Is it clear what the code is doing?
- Do the commit messages explain why the changes were made?
- Are there all the unit tests needed?
- Has all relevant documentation been updated?

### CODE REVIEW

**Comment by lfdebrux** [comment]
That's curious, I haven't been getting the same thing, what version of macOS are you running? I'm on Sonoma 14.2.1.

File: Gemfile.lock [hunk 1]
```diff
@@ -465,6 +465,7 @@ PLATFORMS
   aarch64-linux
   arm64-darwin-21
   arm64-darwin-22
+  arm64-darwin-23
   x86_64-darwin-20
   x86_64-darwin-21
   x86_64-darwin-22
```
Comment (DavidBiddle) [inline Gemfile.lock:471]: Would you mind adding this one as well to save us needing a second PR?
```suggestion
  x86_64-darwin-22
  x86_64-darwin-23
```

**Comment by aliuk2012** [comment]
> That's curious, I haven't been getting the same thing, what version of macOS are you running? I'm on Sonoma 14.2.1.

I'm on the same version as you

**Comment by lfdebrux** [comment]
> > That's curious, I haven't been getting the same thing, what version of macOS are you running? I'm on Sonoma 14.2.1.
> 
> I'm on the same version as you

Ah, I see what's happened here, I upgraded from macOS Ventura to Sonoma recently without rebuilding my version of Ruby, so running Bundler on my machine targeted `darwin-22`. I've rebuilt Ruby now so I'm seeing the same thing now.

**Review by DavidBiddle — APPROVED** [review]

**Pull request merged**